Frontend/src/pages/CSS/login-signup.css

```diff
@@ -1,192 +1,194 @@
-    column-gap: 30px;
-}
-
-.form {
-    max-width: 430px;
-    width: 80%;
-    padding: 30px;
-    border-radius: 6px;
-    background: #fff;
-}
-
-.form.signup {
-    opacity: 0;
-    pointer-events: none;
-}
-
-.forms.show-signup .form.signup {
-    opacity: 1;
-    pointer-events: auto;
-}
-
-.form.login {
-    padding: 1em 1em;
-}
-
-.forms.show-signup .form.login {
-    opacity: 0;
-    pointer-events: none;
-}
-
-header {
-    font-size: 28px;
-    color: #232836;
-    text-align: center;
-}
-
-form {
-    margin-top: 30px;
-}
-
-.form .field {
-    position: relative;
-    height: 50px;
-    width: 100%;
-    margin-top: 20px;
-    border-radius: 6px;
-}
-
-.field input,
-.field button {
-    height: 100%;
-    width: 100%;
-    border: none;
-    font-size: 16px;
-    font-weight: 400;
-    border-radius: 6px;
-}
-
-.field input {
-    outline: none;
-    padding: 0 15px;
-    border: 1px solid#CACACA;
-}
-
-.field input:focus {
-    border-bottom-width: 2px;
-}
-
-.eye-icon {
-    position: absolute;
-    top: 50%;
-    right: 10px;
-    transform: translateY(-50%);
-    font-size: 18px;
-    color: #8b8b8b;
-    cursor: pointer;
-    padding: 5px;
-}
-
-.field button {
-    color: #fff;
-    background-color: #0171d3;
-    transition: all 0.3s ease;
-    cursor: pointer;
-}
-
-.field button:hover {
-    background-color: #016dcb;
-}
-
-.form-link {
-    text-align: center;
-    margin-top: 10px;
-}
-
-.form-link span,
-.form-link a {
-    font-size: 14px;
-    font-weight: 400;
-    color: #232836;
-}
-
-.form a {
-    color: #0171d3;
-    text-decoration: none;
-}
-
-.form-content a:hover {
-    text-decoration: underline;
-}
-
-.line {
-    position: relative;
-    height: 1px;
-    width: 100%;
-    margin: 36px 0;
-    background-color: #d4d4d4;
-}
-
-.line::before {
-    content: "Or";
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    background-color: #fff;
-    color: #8b8b8b;
-    padding: 0 15px;
-}
-
-.media-options a {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-}
-
-a.facebook {
-    color: #fff;
-    background-color: #4267b2;
-}
-
-a.facebook .facebook-icon {
-    height: 30px;
-    width: 30px;
-    color: #0171d3;
-    font-size: 20px;
-    border-radius: 50%;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-color: #fff;
-}
-
-.facebook-icon,
-img.google-img {
-    position: absolute;
-    top: 50%;
-    left: 15px;
-    transform: translateY(-50%);
-}
-
-img.google-img {
-    height: 20px;
-    width: 20px;
-    object-fit: cover;
-}
-
-a.google {
-    border: 1px solid #cacaca;
-}
-
-a.google span {
-    font-weight: 500;
-    opacity: 0.6;
-    color: #232836;
-}
-
-@media screen and (max-width: 400px) {
-    .form {
-        padding: 20px 10px;
-    }
-}
+/* Google Fonts - Poppins */
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap");

```

Comment:
Welldone @Tammy-Ajoko 
Please i would recommend some few changes.
 - Add the google fonts URL in the global CSS, that way you don't have to repeat it in every CSS files